debezium-connector-ibmi/src/main/java/io/debezium/connector/db2as400/As400OffsetContext.java

```diff
@@ -132,8 +132,13 @@ public Schema getSourceInfoSchema() {
         return sourceInfo.schema();
     }
 
-    public void setSourceTime(Instant time) {
-        sourceInfo.setSourceTime(time);
+    public void setSourceTime(Instant timestamp) {
```

Comment:
I think the method either needs a rename or it should be refacored maybe into two. Now its name is very misleading in comparison to what the method does.